### PR TITLE
Tiny general form and table style improvements

### PR DESCRIPTION
**Pull request by annarieger**
* Use `horizontal` form type
* Avoid cropping of long form field labels
* Reintroduce "delete" action on table entities
* Fix placement of "reload" icon in table header

Please review @terrestris/devs

### CODE REVIEW

**Review by hwbllmnn — APPROVED** [review]

File: src/Component/GeneralEntity/GeneralEntityForm/GeneralEntityForm.less [hunk 1]
```diff
@@ -4,4 +4,13 @@
       margin: 0 5px;
     }
   }
+
+  .ant-form-item-label>label {
```
Comment (dnlkoch) [inline src/Component/GeneralEntity/GeneralEntityForm/GeneralEntityForm.less:8]:
```suggestion
  .ant-form-item-label > label {
```

File: src/Component/GeneralEntity/GeneralEntityTable/GeneralEntityTable.tsx [hunk 2]
```diff
@@ -60,19 +66,67 @@ export function GeneralEntityTable<T extends BaseEntity> ({
...
 
     if (cellRendererName === 'DateCell') {
-      return <DisplayField value={text} format="date"/>;
+      return <DisplayField value={text} format="date" />;
     }
 
     return <>{text}</>;
   };
 
-  const tableColumns: ColumnType<T>[] = useMemo(() => {
```
Comment (dnlkoch) [inline src/Component/GeneralEntity/GeneralEntityTable/GeneralEntityTable.tsx:75]: Why did you remove the use of `useMemo`?
